.env.test

```diff
@@ -0,0 +1,5 @@
+DB_USER=postgres
+DB_PASSWORD=sheriff123
+DB_NAME=test
+DB_HOST=localhost
+STORE_TYPE=file
```

Comment:
Change this to db so we are sure db test is passing as well

Reply:
Do you mean I should change the storage from file to dB for the test? Won't that mean that I won't be using the fileRepository again?

And to do that, should I create a new dbTestTRepository or use the one that I am using for dB. Which will require changes in the adapter right?

Reply:
Oh okay